python/feathub/common/protobuf/featureValue.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 The Feathub Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package protobuf;
+
+import public "google/protobuf/timestamp.proto";
+
+message FeatureValue {
+    oneof FeatureValueOneOf {
+        bool none_value = 1;
+        bytes bytes_value = 2;
+        string string_value = 3;
+        int32 int_value = 4;
+        int64 long_value = 5;
+        double double_value = 6;
+        float float_value = 7;
+        bool boolean_value = 8;
+        google.protobuf.Timestamp timestamp_value = 9;
+        Collection vector_value = 10;
+        Collection map_value = 11;
+    }
+}
+
+message Collection {
```

Comment:
Can we make the name/structure of this protobuf more consistent with classes defined in types.py?